diff -up bumblebee-3.2.1/src/bbconfig.h.gcc10fix bumblebee-3.2.1/src/bbconfig.h
--- bumblebee-3.2.1/src/bbconfig.h.gcc10fix	2020-11-23 20:44:15.086455306 -0500
+++ bumblebee-3.2.1/src/bbconfig.h	2020-11-24 10:09:01.654393785 -0500
@@ -102,7 +102,7 @@ enum bb_pm_method {
     PM_VGASWITCHEROO,
     PM_METHODS_COUNT /* not a method but a marker for the end */
 };
-const char *bb_pm_method_string[PM_METHODS_COUNT];
+extern const char *bb_pm_method_string[PM_METHODS_COUNT];
 
 /* String buffer size */
 #define BUFFER_SIZE 1024
diff -up bumblebee-3.2.1/src/bbsecondary.c.gcc10fix bumblebee-3.2.1/src/bbsecondary.c
--- bumblebee-3.2.1/src/bbsecondary.c.gcc10fix	2020-11-24 10:17:18.400121004 -0500
+++ bumblebee-3.2.1/src/bbsecondary.c	2020-11-24 10:18:34.705386310 -0500
@@ -37,6 +37,9 @@
 #include "pci.h"
 #include "module.h"
 
+/* PCI Bus ID of the discrete video card */
+struct pci_bus_id *pci_bus_id_discrete;
+
 /**
  * Substitutes DRIVER in the passed path
  * @param x_conf_file A path to be processed
diff -up bumblebee-3.2.1/src/bbsecondary.h.gcc10fix bumblebee-3.2.1/src/bbsecondary.h
--- bumblebee-3.2.1/src/bbsecondary.h.gcc10fix	2020-11-24 10:13:36.072347949 -0500
+++ bumblebee-3.2.1/src/bbsecondary.h	2020-11-24 10:14:44.916587330 -0500
@@ -30,7 +30,7 @@
 #define XORG_BINARY "/usr/libexec/Xorg.wrap"
 
 /* PCI Bus ID of the discrete video card */
-struct pci_bus_id *pci_bus_id_discrete;
+extern struct pci_bus_id *pci_bus_id_discrete;
 
 /// Start the X server by fork-exec, turn card on if needed.
 bool start_secondary(bool);
diff -up bumblebee-3.2.1/src/switch/switching.c.gcc10fix bumblebee-3.2.1/src/switch/switching.c
--- bumblebee-3.2.1/src/switch/switching.c.gcc10fix	2020-11-24 10:15:23.580721768 -0500
+++ bumblebee-3.2.1/src/switch/switching.c	2020-11-24 10:16:49.827021655 -0500
@@ -31,6 +31,8 @@ struct switching_method switching_method
           switcheroo_on, switcheroo_off}
 };
 
+struct switching_method *switcher = NULL;
+
 /**
  * Enumerates through available switching methods and try a method
  * 
diff -up bumblebee-3.2.1/src/switch/switching.h.gcc10fix bumblebee-3.2.1/src/switch/switching.h
--- bumblebee-3.2.1/src/switch/switching.h.gcc10fix	2020-11-24 10:09:47.067551688 -0500
+++ bumblebee-3.2.1/src/switch/switching.h	2020-11-24 10:12:50.273188708 -0500
@@ -60,10 +60,10 @@ void switcheroo_off(void);
 
 /* number of switchers as defined in switching.c */
 #define SWITCHERS_COUNT 2
-struct switching_method switching_methods[SWITCHERS_COUNT];
+extern struct switching_method switching_methods[SWITCHERS_COUNT];
 
 /* A switching method that can be used or NULL if none */
-struct switching_method *switcher;
+extern struct switching_method *switcher;
 
 struct switching_method *switcher_detect(const char *name, struct switch_info);
 enum switch_state switch_status(void);
